src/babel-config.js

```diff
@@ -19,11 +19,7 @@ module.exports = function(additionalExtensions = ['.ts', '.tsx']) {
     ],
     plugins: [
       'macros',
-      '@babel/plugin-proposal-object-rest-spread',
-      '@babel/plugin-proposal-class-properties',
-      '@babel/plugin-proposal-optional-chaining',
       '@babel/transform-runtime',
-      'babel-plugin-transform-dynamic-import',
```

Comment:
i'm like... pretty sure you can't just remove these. doesn't the static site render fail without them?

Reply:
babel-plugin-transform-dynamic-import should not be used with a bundler [according to the docs](https://babeljs.io/docs/babel-plugin-transform-dynamic-import). Webpack handles dynamic import.

Reply:
The plugin-proposals were added during some upgrades years ago and have been included in @babel-preset since 2020.

Reply:
ok. they were necessary at one point, but i guess its fine now